src/daemon/daemon.js

```diff
@@ -79,7 +79,7 @@ module.exports = async function (opts) {
     logger.info(`[daemon] PeerID is ${id}`)
     logger.info(`[daemon] Repo is at ${ipfsd.path}`)
   } catch (err) {
-    if (!err.message.includes('ECONNREFUSED')) {
+    if (!err.message.includes('ERR_CONNECTION_REFUSED')) {
```

Comment:
Do you remember why this change? 
Did Chromium/Electron change the error code?

Reply:
Good question. I can't find anywhere that indicates that this was changed in either Node.js or Electron. However, this PR does not update Electron, only some IPFS dependencies. I remember updating Node.js a few weeks ago. Maybe that's it?

In addition, I notice that most google results for `ERR_CONNECTION_REFUSED` are quite recent, while for `ECONNREFUSED` are a few years old.

Reply:
I added the check for both instead.